local-cli/bundle/bundleCommandLineArgs.js

```diff
@@ -57,5 +57,9 @@ module.exports = [
     command: 'verbose',
     description: 'Enables logging',
     default: false,
+  }, {
+    command: 'reset-cache',
+    description: 'Removes cached files',
+    default: false
```

Comment:
@chirag04 Yeah, I thought about that and it makes sense in the most cases in which you need the `bundle`. But for `react-native start` it's false and I wanted to keep the behaviour consistent. Adding this to the `react-native-xcode.sh` seems a better practice